admin_manual/maintenance/upgrade.rst

```diff
@@ -2,80 +2,79 @@
-------------------------
+releases increases the risk of errors. Major releases are 8.0, 8.1, and 
+8.2. Point releases are intermediate releases for each major release. For 
+example, 8.0.9 and 8.1.3 are point releases.
 
-The best method for keeping ownCloud on Linux servers current is by 
-configuring your system to use our `Open Build Service 
-<https://download.owncloud.org/download/repositories/8.2/owncloud/>`_ (see 
-:doc:`../installation/linux_installation`); just 
-follow the instructions on oBS for setting up your package manager. Then 
-stay current by using your Linux package manager to upgrade. 
+There are multiple ways to keep your ownCloud server upgraded: with the 
+:doc:`Updater App <update>` (Server Edition only), with your Linux package 
+manager, and by manually upgrading. In this chapter you will learn how to keep 
+your ownCloud installation current with your Linux package manager, and by 
+manually upgrading.
 
 .. note:: Enterprise Subscription customers will use their Enterprise software
-   repositories to install and update their ownCloud installations, rather 
-   than the openSUSE Build Service. Please see    
+   repositories to install ownCloud packages, rather 
+   than the Open Build Service. Then follow the instructions on this page 
+   for completing upgrades. Please see    
    :doc:`../enterprise_installation/linux_installation` for more information.
 
-You should always maintain regular backups (see :doc:`../maintenance/backup`), 
-and make a backup before every upgrade.
+When you are upgrading to a major release, evaluate any third-party apps for 
+compatibility with the upgrade, and then disable them before upgrading. You may 
+re-enable them after the upgrade is completed.
 
-When a new ownCloud release is available you will see a yellow banner in your 
-ownCloud Web interface.
+.. note:: **Downgrading** is not supported and risks corrupting your data! If 
+   you want to revert to an older ownCloud version, make a new, fresh 
+   installation and then restore your data from backup. Before doing this, 
+   file a support ticket (if you have paid support) or ask for help in the 
+   ownCloud forums to see if your issue can be resolved without downgrading.
+
+Upgrade Quickstart
+------------------
+
+The best method for keeping ownCloud on Linux servers current is by configuring 
+your system to use ownCloud's `Open Build Service`_ repository. Then stay 
+current by using your Linux package manager to install fresh ownCloud packages. 
+After installing upgraded packages you must run a few more steps to complete 
+the upgrade. These are the basic steps to upgrading ownCloud:
+
+* Disable all third-party apps.
+* Make a fresh backup (see :doc:`../maintenance/backup`).
+* Install new packages from the ownCloud `Open Build Service`_.
+* Take your ownCloud server out of maintenance mode.
+* Run the upgrade wizard.
+* Log in and apply strong permissions to your ownCloud directories apps.
```

Comment:
Do you mean "your ownCloud apps directories" or "your ownCloud directories and apps"?